src/randomness.rs

```diff
@@ -238,81 +196,3 @@ pub mod testing {
     }
   }
 }
-
-#[cfg(test)]
-mod tests {
-  use super::testing::LocalFetcher;
-  use super::*;
-  use crate::format::*;
-  use crate::internal::*;
-  use httpmock::prelude::*;
-  use serde_json::json;
-
-  #[test]
-  fn test_ppoprf_fetching() {
```

Comment:
This test is essentially checking that the output of `eval` on unblinded points, is the same as the output of the full protocol on blinded points. Even if we want to get rid of the HTTP mock stuff, then we need some way of still making sure this holds this to check that the client blinding code is correct.

Reply:
Do the doctests in lib.rs and tests in api.rs already cover this?

Reply:
As far as I know, no. The tests in api.rs and lib.rs only really check that the nested star functionality passes. But the tests here in randomness.rs check that the randomness outputs that clients receive (internally during nested star) also work. Clearly these other tests wouldn't pass if this was broken. However, having these tests allows us to identify whether randomness generation should be the problem or not.

Reply:
The other tests use the `LocalFetcher`/PPRF. Is this not sufficient for a pseudo-end-to-end test? It seems that any other randomness related functionality should have tests present in the ppoprf crate or star-andsrv. Sorry if I'm misunderstanding, I'm just unsure of what additional test I could add here. If you have a suggestion I'll be happy to add a test in.

Reply:
Sorry for the long wait here. I guess it was more just to make sure that the extra logic that is added in this module doesn't somehow misinterpret the voprf exchange. However, looking over the code it seems like the differences are marginal and given that the LocalFetcher is used outside then maybe this is enough after all.